content/pages/meetup/2023/0726-nycu.rst

```diff
@@ -0,0 +1,85 @@
+========================================
+Meetup 2023 July 26th in NYCU
+========================================
+
+:date: 2023-07-20 13:40
+:url: meetup/2023/0726-nycu
+:save_as: meetup/2023/0726-nycu.html
+
+Date
+-----
+
+* Date: July 26th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:30 Free discussion
+* 19:40-20:10 Subject (sharing): Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis
+* 20:10-21:30 Subject (discussion): Preparations for August scisprint  
+
+|
+
+Subjects
+------------------
+
+* **Sharing: Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis**
+
+    Speaker : Nian-Cheng - A graduate student in NYCU Institute of Artificial Intelligence Innovation.
+    
+    In this topic, I leveraged data from depth cameras for AI model data processing and introduces the essential tools I utilized for this purpose.
+
+* **Discussion: Preparations for August scisprint**
+
+    Additionally, we will discuss the following items for preparing August scisprint. 
+
+    * Task arrangement for volunteers
+    * Promotion plan for August scisprint
+    * Preparations for career conversation
+
+|
+
+Sign up
+------------
+
+The event is free. Please register on `sign up <https://sciwork.kktix.cc/events/meetup-20230726>`__.
```

Comment:
The text could be written as `The event is free. Please register at kktix` ,which may be more clear.

Reply:
Agree, something like:

```rst
The event is free. Please register at `kktix <https://sciwork.kktix.cc/events/meetup-20230726>`__.
```